shuup/admin/modules/orders/views/edit.py

```diff
@@ -340,6 +340,7 @@ def handle_customer_details(self, request):
                 "name": customer.full_name,
                 "phone_no": customer.phone,
                 "email": customer.email,
+                "tax_number": customer.tax_number if isinstance(customer, CompanyContact) else "",
```

Comment:
I would prefer `getattr(customer, "tax_number", "")` instead of isinstance check.